src/@next/IndexTable/IndexTableStyle.ts

```diff
@@ -1242,9 +1242,9 @@ export const StyledIndexTable: any = createGlobalStyle`
   .Polaris-IndexTable__Table--unselectable.Polaris-IndexTable__Table--sticky
     .Polaris-IndexTable__TableCell:first-child {
     left: 0;
-    background-color: var(--p-surface);
     z-index: var(--pc-index-table-sticky-cell);
     position: sticky;
+    box-shadow: inset 0 -0.0625rem 0 #d4d5d8;
```

Comment:
Is this color not part of color token? Also applicable for other colors

Reply:
ohh it's part! let me update to use the token